src/const.js

```diff
@@ -162,7 +162,9 @@ export const MESSAGES_JSON = 'messages.json';
 export const LOCALES_DIRECTORY = '_locales';
 
 // This is a string, since it has to be matched globally on a message string.
-export const MESSAGE_PLACEHOLDER_REGEXP = '\\$([a-z0-9_@]+)\\$';
+// This should match
+// https://searchfox.org/mozilla-central/rev/3abf6fa7e2a6d9a7bfb88796141b0f012e68c2db/toolkit/components/extensions/ExtensionCommon.jsm#1711
+export const MESSAGE_PLACEHOLDER_REGEXP = '\\$([a-zA-Z0-9_@]+)\\$';
```

Comment:
I hadn't included the uppercase letters because I passed the ignore case param to the RegExp constructor, but I guess better safe than sorry ;)